ark/utils/deepcell_service_utils_test.py

```diff
@@ -7,7 +7,7 @@
 from ark.utils.deepcell_service_utils import create_deepcell_output
 
 
-def mocked_run_deepcell(input_dir, output_dir, host, job_type, scale):
+def mocked_run_deepcell(input_dir, output_dir, host, job_type, scale, timeout):
```

Comment:
Can add some simple checks for ```_zip_run_extract```
e.g.  for 210 images, make sure first zip has 100, second zip has 100, third zip has 10. 
Make sure first zip contains first 100 images. 
Make sure changing zip_size actually changes number of images included. 

Can change the default to be like 5 or 2 or something to make previous tests faster.